legend-depot-store-mongo/src/main/java/org/finos/legend/depot/store/mongo/BaseMongo.java

```diff
@@ -147,6 +147,13 @@ public List<T> getAllStoredEntities()
         return result;
     }
 
+    public List<T> getStoredEntitiesByPage(int page, int pageSize)
```

Comment:
pageSize is confusing. The ProjectsService uses pageSize to indicate the # of projects. Here it is the # of entities ?
Are there 2 notions of pageSize ?

Reply:
no, this `entity` refers to `mongo entity`